src/main/java/com/staxrt/tutorial/controller/PostController.java

```diff
@@ -0,0 +1,73 @@
+package com.staxrt.tutorial.controller;
+/* https://www.baeldung.com/spring-data-jpa-multiple-databases */
+
+import com.staxrt.tutorial.exception.ResourceNotFoundException;
+import com.staxrt.tutorial.model.Post;
+import com.staxrt.tutorial.model.User;
+import com.staxrt.tutorial.repository.PostRepository;
+import com.staxrt.tutorial.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@RestController
+@RequestMapping("/api/v2")
```

Comment:
A w wersji 1 co było? Czemu na starcie już wystawiasz api w wersji 2?

Reply:
Request mapping dla post controller i user controller musi być pod innym linkiem chciałem by user oraz post byli nasluchiwani na tym samym /API/V1 ale nie mogę tak. Lepiej jest użyć 2 controllerow czy 1 w takim przypadku? Chciałem by każdy model miał osobnego controllera ale nie wiem czy to jest najlepsze rozwiązanie

Reply:
hmm, nie no dwa controllery są ok, natomiast mimo to musi się dać zrobić wszystko pod jednym zasobem api. Musisz doczytać sobie na https://docs.spring.io/spring-framework/docs/current/reference/html/web.html#mvc

Reply:
review